### PR TITLE
Hiding Data Structures moved to positioning

### PR DESCRIPTION
**Pull request by mayanksharma3**
### Clubhouse Ticket
[ch328](https://app.clubhouse.io/manim-dsl/story/328/resolve-bug-about-hiding-data-structures-for-variables-with-same-name)

### Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue) 

### Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue) :bug
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### How Has This Been Tested?

Please describe tests added as well.

### Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by iz0825 — DISMISSED** [review]
looks good to me! maybe wait until @adityagoel4512 checks too?

File: src/main/kotlin/com/manimdsl/stylesheet/Stylesheet.kt [hunk 1]
```diff
@@ -149,6 +146,14 @@ class Stylesheet(private val stylesheetPath: String?, private val symbolTableVis
     fun getDisplayNewLinesInCode(): Boolean = stylesheet.displayNewLinesInCode
 
     fun getTabSpacing(): Int = stylesheet.tabSpacing
+
+    fun renderDataStructure(identifier: String): Boolean {
+        return if (stylesheet.positions.containsKey(identifier)) {
```
Comment (adityagoel4512) [inline src/main/kotlin/com/manimdsl/stylesheet/Stylesheet.kt:151]: rest of logic looks fine could be worth replacing this with && though

Comment (adityagoel4512) [reply src/main/kotlin/com/manimdsl/stylesheet/Stylesheet.kt:151]: return !stylesheet.positions.containsKey(identifier) || stylesheet.positions[identifier]!!.height != 0.0

Comment (adityagoel4512) [reply src/main/kotlin/com/manimdsl/stylesheet/Stylesheet.kt:151]: nitpick up to you to incorporate or not

**Review by iz0825 — APPROVED** [review]

**Pull request merged**